CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
+
+## [Unreleased]
+Created repo with initial commit (@mbbroberg)
```

Comment:
these need to go under an `### Added` header

Reply:
D'oh... I should know better by now!
Thanks 👍